addons/web/static/src/xml/graph.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <div t-name="web.GraphRenderer" owl="1">
+        <div class="o_graph_renderer">
+            <label t-if="props.title"><t t-esc="props.title"/></label>
+            <t t-if="noContentHelperData" t-call="web.NoContentHelper">
+                <t t-set="title" t-value="noContentHelperData.title"/>
+                <t t-set="description" t-value="noContentHelperData.description"/>
+            </t>
+            <div t-else="" class="o_graph_canvas_container" t-attf-style="{{props.isEmbedded ? '' : 'position: absolute;'}}">
```

Comment:
maybe add conditional className o_embedded_graph or something, and the rule in scss?

Reply:
should not be there at all